web/src/buddy_list.js

```diff
@@ -10,10 +10,10 @@ import * as padded_widget from "./padded_widget";
 import * as scroll_util from "./scroll_util";
 
 class BuddyListConf {
-    container_sel = "#buddy-list-users-matching-view";
-    scroll_container_sel = "#buddy_list_wrapper";
-    item_sel = "li.user_sidebar_entry";
-    padding_sel = "#buddy_list_wrapper_padding";
+    container_selector = "#buddy-list-users-matching-view";
+    scroll_container_selector = "#buddy_list_wrapper";
+    item_selector = "li.user_sidebar_entry";
+    padding_selector = "#buddy_list_wrapper_padding";
```

Comment:
Extracted this first commit, which lgtm, as https://github.com/zulip/zulip/pull/27807.